cloud-controller-manager/do/loadbalancers.go

```diff
@@ -660,6 +660,10 @@ func buildForwardingRules(service *v1.Service) ([]godo.ForwardingRule, error) {
 		httpsPorts = append(httpsPorts, defaultSecurePort)
 	}
 
+	if getRedirectHTTPToHTTPS(service) && len(httpsPorts) == 0 {
```

Comment:
Should this include `http2Ports`?
```go
if getRedirectHTTPToHTTPS(service) && len(httpsPorts) + len(http2Ports) == 0 {
```

Reply:
I was under the impression that HTTP/2 wouldn't be considered by the LB product, but turns out that it is.

I updated the PR accordingly. Also made a small addendum to the service annotation docs.